client/state/stats/lists/selectors.js

```diff
@@ -101,3 +103,23 @@ export const getSiteStatsMaxPostsByDay = createSelector(
 	}
 );
 
+/**
+ * Returns a number representing the posts made during a day for a given query
+ *
+ * @param  {Object}  state    Global state tree
+ * @param  {Number}  siteId   Site ID
+ * @param  {Object}  query    Stats query object
+ * @return {?Number}          Max number of posts by day
+ */
+export const getSiteStatsPostsCountByDay = createSelector(
```

Comment:
Do we need to memoize this given how simple the logic is and since `getSiteStatsPostStreakData` is already memoized?